src/constants.h

```diff
@@ -95,11 +101,11 @@ namespace KeyFinder {
     SCALE_MINOR
   };
 
-  double getFrequencyOfBand(unsigned int band);
-  double getLastFrequency();
+  KEYFINDER_DECLSPEC double getFrequencyOfBand(unsigned int band);
```

Comment:
Should the enums above be exported too?

Reply:
No, enums are included via header files.

Reply:
okay